### PR TITLE
fixing loading indicator error messaging for document import

### PR DESCRIPTION
**Pull request by dharmamike**
fixes https://www.pivotaltracker.com/story/show/96125716

### CODE REVIEW

**Comment by rheinwein** [comment]
implicit :+1:

File: test/spec/controllers/document-import.js [hunk 1]
```diff
@@ -272,6 +273,11 @@ describe('Controller: DocumentImportCtrl', function () {
           DocumentImportCtrl.importPastedContent(docImport.raw);
           expect(scope.yamlDocument.loadFailure).toBe(true);
         });
+
+        it('sets yamlDocument.loadFailure true', function () {
+          DocumentImportCtrl.importPastedContent(docImport.raw);
+          expect(scope.setLoading).toHaveBeenCalledWith(false);
```
Comment (alexwelch) [inline test/spec/controllers/document-import.js]: is ```setLoading``` already tested somewhere? I was thrown off by the test description, but assume that it's setting ```loadFailure``` inside that function call

**Comment by alexwelch** [comment]
:+1:

File: test/spec/controllers/document-import.js (hunk 1 above)
Comment (dharmamike) [inline test/spec/controllers/document-import.js]: Good catch.  The description is wrong.

Comment (dharmamike) [inline test/spec/controllers/document-import.js]: fixed.

**Pull request merged**